lib/platform/github/index.ts

```diff
@@ -1583,8 +1586,9 @@ export async function getPrFiles(prNo: number) {
   return files.map((f: { filename: string }) => f.filename);
 }
 
-export async function updatePr(prNo: number, title: string, body?: string) {
+export async function updatePr(prNo: number, title: string, rawBody?: string) {
   logger.debug(`updatePr(${prNo}, ${title}, body)`);
+  const body = rawBody ? hostRules.sanitize(rawBody) : null;
```

Comment:
`hostRules.sanitize` already checks for `null`

Reply:
`body` string should be removed from log.

do we need to rename params? Can we simply move the sanitize to the if below?